src/examples/using-with-cmake/CMakeLists.txt

```diff
@@ -51,19 +51,19 @@
 #  mkdir build
 #  cd build
 #  cmake \
-#   -DASCENT_DIR={ascent install path}   \
-#   -DCONDUIT_DIR={conduit install path} \
+#   -DAscent_DIR={ascent install path}   \
+#   -DConduit_DIR={conduit install path} \
 #   -DVTKH_DIR={vtkh install path}          
 #   ../
 # make
 # ./ascent_render_example
 #
-# If run in sub directory below using-with-cmake in an ascent install, 
-# ASCENT_DIR will be defaulted to ../../..
+# In order to run directly in a sub directory below using-with-cmake in an ascent install, 
+# set Ascent_DIR to ../../..
 # 
 #  mkdir build
 #  cd build
-#  cmake ..
+#  cmake .. -DAscent_DIR=../../..
 #  make
 #  ./ascent_render_example
 #
```

Comment:
I feel this block would be more visible in a README.md or INSTALL.md file in the same directory, but that's taste.

Reply:
this gets pulled into the docs here:

https://ascent.readthedocs.io/en/latest/QuickStart.html#using-ascent-in-your-project

which is where I expect most folks to discover it.  Also folks cut-and-paste and hack away, so having it in the file they are editing is useful.

Reply:
Then it's totally cool I think :)